js/bootstrap-datepicker.js

```diff
@@ -2034,3 +2038,13 @@
 	});
 
 }));
+
+function deprecate(msg) {
```

Comment:
I've pushed the change so you can see the test result on travis and how I declared the function!